tests/model_hub_tests/torch_tests/test_torchvision_models.py

```diff
@@ -56,7 +56,8 @@ class TestTorchHubConvertModel(TestConvertModel):
     def setup_class(self):
         self.cache_dir = tempfile.TemporaryDirectory()
         # set temp dir for torch cache
-        torch.hub.set_dir(str(self.cache_dir.name))
+        if os.environ.get('USE_SYSTEM_CACHE', 'True') == 'False':
+            torch.hub.set_dir(str(self.cache_dir.name))
```

Comment:
I remember that we have a logic that deletes downloaded models. Should we correct it as well?

Reply:
In the current solution it will clear empty directory, which is fine, but if we want to provide cache directory as @ilya-lavrenov suggests, we would need to do something with cleaning